### PR TITLE
[_654] further documentation of script in usage help

### PR DESCRIPTION
**Pull request by d-w-moore**
I think this rounds out what we need as documentation in the script. b4599282a362ebf4ca069342ff26fb097585bef4 has now  also provided docstrings for the public-facing free functions of this module

### CODE REVIEW

File: irods/client_init.py
Comment (korydraughn) [inline irods/client_init.py:69]: files -> file

**Review by korydraughn — APPROVED** [review]
Looks good.

Please wait for one more approval before pounding.

File: irods/client_init.py
Comment (d-w-moore) [reply irods/client_init.py]: Done

**Review by trel — APPROVED** [review]
looks good

**Comment by d-w-moore** [comment]
time to \#?

**Comment by trel** [comment]
yes approved - please #

**Comment by d-w-moore** [comment]
Squashed and pounded

**Review by alanking — APPROVED** [review]

**Pull request merged**
